test/templates.test.js

```diff
@@ -36,14 +34,15 @@ const checkTemplateStructureAndRun = async (actorName) => {
 
     // Check if actor was created without errors
     expect(console.log.args.map((arg) => arg[0])).not.toContain('Error:');
+};
 
+const checkTemplateRun = async (actorName) => {
     process.chdir(actorName);
     spawnSync('apify', ['run']);
     process.chdir('../');
-
     // Check if actor run without errors
     expect(console.log.args.map((arg) => arg[0])).not.toContain('Error:');
-};
+}
```

Comment:
nit (i guess we dont lint the tests?)
```suggestion
};
```